doc/ref_model/chapters/chapter07.md

```diff
@@ -172,6 +171,12 @@ The platform supports the workload, and in effect controls access to the workloa
 * Ensure that all the platform's components (including hypervisors, VMs, etc.) are kept up to date with the latest patch.
 * In order to tightly control access to resources and protect them from malicious access and introspection, Linux Security Modules such as SELinux should be used to enforce access rules.
 
+**Vulnerability Management**
+* Security defect must be reported.
```

Comment:
```suggestion
* Security defects must be reported.
```

Reply:
@pgoyal01 Thanks, I have added your suggestion into the last commit modifying the whole paragraph. Please review